tests/functional/helpers.js

```diff
@@ -9,31 +9,28 @@ export const getUri = uri => browser.options.baseUrl + uri
 export const getRandomString = length => sampleSize('ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789', length).join('')
 
 export const startCensusQuestionnaire = (schema, sexualIdentity = false, region = 'GB-ENG', language = 'en') => {
-  devPage.open()
-    .setUserId(getRandomString(10))
-    .setCollectionId(getRandomString(10))
-    .setSchema(schema)
-    .setRegionCode(region)
+  generateToken(schema, getRandomString(10), getRandomString(10), null, null, region, language, sexualIdentity)
+    .then(function(token) {
+      browser.url('http://localhost:5000/session?token=' + token)
+    })
 
-  devPage.setLanguageCode(language)
-
-  if (sexualIdentity) {
-    devPage.checkSexualIdentity()
-  }
-
-  devPage.submit()
+  browser.pause(1000) // Shudder!!!
```

Comment:
This will go away as part of my work on https://trello.com/c/7H5ReU7v/1095-refactor-functional-test-page-objects